TipCatDotNet.Api/Infrastructure/Validations.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace TipCatDotNet.Api.Infrastructure
+{
+    public static class Validations
+    {
+        public static bool IsEmail(string param, bool nullable = true)
```

Comment:
Did we need it? We always may confirm email by sending something to it

Reply:
This was only added to check the data entry is correct